### PR TITLE
media: Fix build error on linux

### PR DESCRIPTION
**Pull request by gcjjyy**
MediaFramework should be built on TizenRT and Linux both.
Linux build is more strict than TizenRT.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [7b0af70d32a90f186a247aa600a81f2f1119fc7c] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [7b0af70d32a90f186a247aa600a81f2f1119fc7c] - Code Rule Check Result:
framework/src/media/MediaQueue.h:35: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
framework/src/media/MediaQueue.h:35: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
framework/src/media/MediaQueue.h:36: ERROR: [BRC_M_FTN] open brace '{' following function declarations go on the next line
framework/src/media/MediaQueue.h:36: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaQueue.h:36: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] need consistent spacing around '>>' (ctx:VxW)
framework/src/media/MediaQueue.h:47: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:48: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Comment by seinfra** [comment]
Target : [b73fd55436ea01edff46f5642590b3626cde269f] -

**Comment by seinfra** [comment]
Target : [b73fd55436ea01edff46f5642590b3626cde269f] -

**Comment by seinfra** [comment]
Target : [c0e20d733b15983db96576dc58d8cde46066ea18] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [c0e20d733b15983db96576dc58d8cde46066ea18] - Code Rule Check Result:
framework/src/media/MediaQueue.h:35: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
framework/src/media/MediaQueue.h:35: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
framework/src/media/MediaQueue.h:36: ERROR: [BRC_M_FTN] open brace '{' following function declarations go on the next line
framework/src/media/MediaQueue.h:36: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaQueue.h:36: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] need consistent spacing around '>>' (ctx:VxW)
framework/src/media/MediaQueue.h:47: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:48: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

File: framework/src/media/FileInputDataSource.cpp [hunk 1]
```diff
@@ -16,11 +16,20 @@
...
 #include "Decoder.h"
 
+#ifndef OK
+#define OK (0)
+#endif
+
+#ifndef EOF
+#define EOF (-1)
+#endif
+
```
[the remark this answers is not in the document]
Comment (gcjjyy) [reply framework/src/media/FileInputDataSource.cpp]: Awesome. I'll fix!

**Comment by seinfra** [comment]
Target : [9f1408d9d06522cb3349357f7b922eb515bfd030] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9f1408d9d06522cb3349357f7b922eb515bfd030] - Code Rule Check Result:
framework/src/media/MediaQueue.h:35: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
framework/src/media/MediaQueue.h:35: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
framework/src/media/MediaQueue.h:36: ERROR: [BRC_M_FTN] open brace '{' following function declarations go on the next line
framework/src/media/MediaQueue.h:36: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaQueue.h:36: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:37: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:42: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaQueue.h:46: ERROR: [SPC_M_OPR] need consistent spacing around '>>' (ctx:VxW)
framework/src/media/MediaQueue.h:47: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaQueue.h:48: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Pull request merged**